template/atlas/aal/ROI_MNI_V4.txt

```diff
@@ -88,24 +88,24 @@ T2AG	Temporal_Pole_Mid_L	8211
 T2AD	Temporal_Pole_Mid_R	8212
 T3G	Temporal_Inf_L	8301
 T3D	Temporal_Inf_R	8302
-CERCRU1G	Cerebelum_Crus1_L	9001
-CERCRU1D	Cerebelum_Crus1_R	9002
-CERCRU2G	Cerebelum_Crus2_L	9011
-CERCRU2D	Cerebelum_Crus2_R	9012
-CER3G	Cerebelum_3_L	9021
-CER3D	Cerebelum_3_R	9022
-CER4_5G	Cerebelum_4_5_L	9031
-CER4_5D	Cerebelum_4_5_R	9032
-CER6G	Cerebelum_6_L	9041
-CER6D	Cerebelum_6_R	9042
-CER7BG	Cerebelum_7b_L	9051
-CER7BD	Cerebelum_7b_R	9052
-CER8G	Cerebelum_8_L	9061
-CER8D	Cerebelum_8_R	9062
-CER9G	Cerebelum_9_L	9071
-CER9D	Cerebelum_9_R	9072
-CER10G	Cerebelum_10_L	9081
-CER10D	Cerebelum_10_R	9082
+CERCRU1G	Cerebellum_Crus1_L	9001
+CERCRU1D	Cerebellum_Crus1_R	9002
+CERCRU2G	Cerebellum_Crus2_L	9011
+CERCRU2D	Cerebellum_Crus2_R	9012
+CER3G	Cerebellum_3_L	9021
+CER3D	Cerebellum_3_R	9022
+CER4_5G	Cerebellum_4_5_L	9031
+CER4_5D	Cerebellum_4_5_R	9032
+CER6G	Cerebellum_6_L	9041
+CER6D	Cerebellum_6_R	9042
+CER7BG	Cerebellum_7b_L	9051
+CER7BD	Cerebellum_7b_R	9052
+CER8G	Cerebellum_8_L	9061
+CER8D	Cerebellum_8_R	9062
+CER9G	Cerebellum_9_L	9071
+CER9D	Cerebellum_9_R	9072
+CER10G	Cerebellum_10_L	9081
+CER10D	Cerebellum_10_R	9082
 VER1_2	Vermis_1_2	9100
 VER3	Vermis_3	9110
 VER4_5	Vermis_4_5	9120
```

Comment:
this seems unrelated to the bug fix.

Reply:
That's the "fix typo in AAL atlas labels" part of the PR.... Ok, it accidentally got selected when pushing this PR, but it needed to happen anyway, so I left it in. :-)

Reply:
@sarangnemo this tells me that merging of PR should not be automatic and done by the person that opens the PR. No need to enforce this but it's good practice to ask for quick review before merging code to master.

Reply:
Normally wouldn't do that, but got a (apparently premature!) vacation message from Robert and didn't realize you'd be able to review this here. I acknowledge AAL label update should have been a separate PR, but was consciously submitted in this PR in lieu of digging through git documentation to deselect it.